lib/maths/COutliers.cc

```diff
@@ -556,23 +556,26 @@ void CEnsemble<POINT>::CScorer::add(const TMeanVarAccumulator2Vec& logScoreMomen
     // of benchmark sets to make this effective a priori. We also account for the
     // fact that the scores will be somewhat correlated.
 
+    // We lower bound the coefficient of variation (to 1e-4) of the log-normal we
+    // use to compute the score c.d.f.
+    static const double MINIMUM_VARIANCE{CTools::fastLog(1.0 + 1e-8)};
```

Comment:
It might be better to put this outside the method.  It's a non-trivial static initialisation, so will automatically get protected by some locking mechanism on the compilers that support that.  So putting it in the method will cause some locking overhead each time the method is called.

Alternatively, is `CTools::fastLog(1.0 + 1e-8)` so close to `1e-8` that incorporating the call to `CTools::fastLog()` is unnecessary and the line can become simply `static const double MINIMUM_VARIANCE{1e-8}`?

Reply:
I could use the approximation. I left it like this so it is clear(er) where the value comes from, but maybe I could just add to the comment. This shouldn't be a bottleneck, but better not to be needlessly slow.